src/sentry/static/sentry/app/views/releases/detail/utils.tsx

```diff
@@ -1,24 +1,72 @@
 import {Location} from 'history';
 import pick from 'lodash/pick';
+import {browserHistory} from 'react-router';
 
 import {Client} from 'app/api';
-import {CommitFile, Commit, FilesByRepository, Repository} from 'app/types';
+import {
+  CommitFile,
+  Commit,
+  FilesByRepository,
+  Repository,
+  ReleaseStatus,
+} from 'app/types';
 import {t} from 'app/locale';
 import {URL_PARAM} from 'app/constants/globalSelectionHeader';
+import {
+  addErrorMessage,
+  addLoadingMessage,
+  addSuccessMessage,
+} from 'app/actionCreators/indicator';
 
 export type CommitsByRepository = {
   [key: string]: Commit[];
 };
 
-export const deleteRelease = (orgId: string, version: string) => {
+export const archiveRelease = async (orgId: string, version: string) => {
```

Comment:
Should these be in `app/actionCreators/release` instead? Does the release store need to be updated when releases are archived/unarchived?